src/Database/PostgreSQL/PQTypes/Checks/Util.hs

```diff
@@ -85,6 +86,26 @@ checkNames prop_name = mconcat . map check
           , ")."
           ]]
 
+checkPresence :: RawSQL ()
```

Comment:
`checkPKPresence`? Also needs Haddock comments for both the top-level entity and the parameters.

Reply:
Addressed. Fixing tests now.